templates/compose/sequin.yaml

```
@@ -10,7 +10,7 @@ services:
     image: sequin/sequin:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.
```suggestion
    image: sequin/sequin:v0.13.18 # Released on Oct 6, 2025
```